### PR TITLE
I18N: Update translation information

### PR DESCRIPTION
**Pull request by lotharsm**
I updated the residualvm.pot template file from source via the `update-translation` command, updated the German GUI translation and rebuilt the translations.dat file to make the changes available in the GUI.

### CODE REVIEW

File: po/residualvm/de_DE.po [hunk 1]
```diff
@@ -7,86 +7,93 @@ msgid ""
...
-#: gui/options.cpp:1694
+#: gui/options.cpp:1756
+msgid ""
+"When starting a game, change the GUI language to the game language.That way, "
+"if a game uses the ResidualVM save and load dialogs, they are in the same "
+"language as the game."
+msgstr ""
+"Die Sprache der Men�f�hrung wird beim Starten des Spiels auf die Sprache des "
+"Spiels gestellt. Damit ist sichergestellt, dass bei der Verwendung der "
+"ResidualVM-eigenen \"Speichern und Laden\"-Funktionailit�t gleiche Sprachen "
```
Comment (raziel-) [inline po/residualvm/de_DE.po]: Typo...Funktionalität

Comment (lotharsm) [reply po/residualvm/de_DE.po]: Thanks, good catch!

**Comment by lotharsm** [comment]
Weird... despite having the translations.dat regenerated, it seems that only some strings specific to ResidualVM are not translated on my end even though the vast majority is... For example the "Aspect ratio correction" text as well as the description of the main buttons in the GUI remains untranslated...

Is it possible that somehow not all ResidualVM specific strings are merged into the translations.dat file?

File: po/residualvm/de_DE.po [hunk 2]
```diff
@@ -7,86 +7,93 @@ msgid ""
...
+"ResidualVM-eigenen \"Speichern und Laden\"-Funktionalit�t gleiche Sprachen "
+"verwendet werden."
+
+#: gui/options.cpp:1770
+msgid ""
+"Use the native system file browser instead of the ResidualVM one to select a "
+"file or directory."
+msgstr ""
+"Verwende zur Auswahl einer Datei oder eines Verzeichnisses den "
+"betriebssystemeigenen Dateimanager anstelle der ResidualVM-eigenen "
```
Comment (raziel-) [inline po/residualvm/de_DE.po]: Typo...des (anstatt der)
Except the "Auswahlmenüs" later on are meant to be majority, but then it wouldn´t fit with the earlier "den ... Dateimanager" as this is single aswell?
English seems to be single form in both cases.

German:
Sollte das nicht "des ... Auswahlmenüs" heißen?
Außer die "Auswahlmenüs" sind als Mehrzahl gedacht, dann aber passt das vorherige "den ... Dateimanager" nicht, weil der auch Einzahl ist?
Im Original sind beides Einzahl, soweit ich das verstehe.

Comment (lotharsm) [reply po/residualvm/de_DE.po]: Stimmt, ist tatsächlich etwas inkonsistent, schließlich ist der Dateimanager das einzige "Betriebssystem-Auswahlmenü". Passe ich dann auch gleich in ScummVM an, da ist es genauso "falsch".

**Comment by Botje** [comment]
Is this okay for merging? I have no way of verifying the German translations.

**Comment by lotharsm** [comment]
Regarding the translation itself, I would consider it ready to get merged. The only thing that's left over is to check why the translations are not picked up correctly for the ResidualVM-specific strings.

**Comment by lotharsm** [comment]
I'd like to bring this back to everyone's attention.

The translation itself is correct in it's current stage, but I can't figure out why create_translations isn't processing the files in ./po/residualvm, so the translation of the strings specific to ResidualVM are not translated at the moment...

**Comment by Botje** [comment]
Can you try again (and rebase if necessary) with the changes in that PR? It works for me ...

**Comment by lotharsm** [comment]
Yep, I can confirm it's working!

**Comment by Botje** [comment]
Okay. Merged. Can you rebuild translations.dat and append it to this commit?

**Comment by lotharsm** [comment]
I appended the new translations.dat via 9b6df6e.

**Pull request merged**